CHANGES.md

```diff
@@ -23,7 +23,7 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
 
- * The deprecated functions EVP_PKEY_get0(), EVP_PKEY_get0_RSA(),
```

Comment:
Could you please add a separate CHANGES entry for the EVP_PKEY_get0() change of behavior? I.e., it now returns NULL for non-legacy keys.

Reply:
Done